src/cow-react/modules/limitOrders/containers/LimitOrdersMenuUpdater/index.tsx

```diff
@@ -0,0 +1,7 @@
+import { useParameterizeLimitOrdersInMenu } from '../../hooks/useParameterizeLimitOrdersInMenu'
```

Comment:
Path?

Reply:
I don't remember exactly but seems like we discussed with @anxolin that we should use relative paths in the module scope. I don't have a strong opinion about it.